### PR TITLE
v2.1.0

### PR DESCRIPTION
**Pull request by StormiFire**
- Added `ComponentCreated` event ✨
- Fixed some sentence formations 🤓
- New versioning ✔
  - Follow the top of https://semver.org/ for what the new version protocol is.
  - If I'm correct, unless you patch something, the version should stay the same (you should only bump it once per release even if you've added more than one thing?). Otherwise it should bump to **2.1.1**.
  - This makes it a bit more complicated to figure out milestones.

### CODE REVIEW

**Comment by ysfchn** [comment]
So is it targeting a new release or just fixes the version code/number? If you mean a new version, then I'm not sure about releasing a new version for adding only one block.

File: src/com/yusufcihan/DynamicComponents/DynamicComponents.java [hunk 1]
```diff
@@ -583,15 +598,12 @@ public void SetProperty(Component component, String name, Object value) {
...
-        Supported values are; "string", "boolean", "integer" and "float". For other values, you should use
-        Any Component blocks.
+        Set multiple properties of a component by typing its property name and value. It behaves like a Setter property block. It can be also used to set properties that only exists in Designer. Supported values are; "string", "boolean", "integer" and "float". For other values, you should use "Any Component" blocks.
 
 
         -- Parameters --
-        Component component            : The component that will be modified.
-        String properties                    : Names and values of the properties.
+        Component component - The component that will be modified.
+        Dictionary properties - Names and values of the properties.
```
Comment (ysfchn) [inline src/com/yusufcihan/DynamicComponents/DynamicComponents.java:606]: I see you changed the parameter description in the comment for the `SetProperties` block, however, in the other blocks' comments it is still written as:
```
-- Parameters --
AndroidViewComponent in        : To specify where component will be created in.
```

Comment (StormiFire) [reply src/com/yusufcihan/DynamicComponents/DynamicComponents.java:606]: It shouldn't really make a difference since it's an internal change.

Comment (ysfchn) [reply src/com/yusufcihan/DynamicComponents/DynamicComponents.java:606]: Ok then, I just asked out of curiosity.

**Review by ysfchn — APPROVED** [review]
LGTM

**Pull request merged**